From 2f3b5812536c103efb13924e2668fd725d3aa237 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Sat, 24 Nov 2018 02:56:51 +0900
Subject: [PATCH 3/3] makefile: update Makefile for testing DPDK

This change is to update `T` option from `x86_64-ivshmem-linuxapp-gcc`
which is already deprecated in `Makefile.spp`.

`Makefile.spp` is to compile DPDK with config `*RTE_APP_TEST=y` and
`*RTE_TEST_PMD=y` for testing.

  $ make -f Makefile.spp  # compile DPDK for testing

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 Makefile.spp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.spp b/Makefile.spp
index ec6bab3..cc8d49f 100644
--- a/Makefile.spp
+++ b/Makefile.spp
@@ -29,7 +29,7 @@
 #   (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #   OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-T ?= x86_64-ivshmem-linuxapp-gcc
+T ?= x86_64-native-linuxapp-gcc
 
 all:
 	make -C $(RTE_SDK) T=$(T) config
-- 
2.7.4

